### DISCUSSION TITLE
CPU Temperature

### DISCUSSION THREAD

**Discussion by glebaccon** [dougiteixeira/proxmoxve#180] [Feature request] [answered]
Hello,

It could be very useful to add CPU temperature.

Thanks for your work.

**Comment by dougiteixeira** [comment, marked as answer]
Duplicated with #117.
The Proxmox API does not have this feature available. So it is not possible to include.

**Comment by mZ738** [comment]
Hardware readings are in since 5.0.0: temperatures, fan speeds, voltages and power from `lm-sensors`, through [PVE-mods](https://github.com/Meliox/PVE-mods) on the node — the integration reads them from the node status the mod extends, no extra API calls. Both the current `node_info` package and the legacy script are recognised. How to set it up, which chips are classified how, and why readings come and go with the mod's on-demand collector: [Hardware sensors](https://github.com/dougiteixeira/proxmoxve/blob/main/docs/hardware-sensors.md). 5.1.0 ([release notes](https://github.com/dougiteixeira/proxmoxve/releases/tag/5.1.0)) is out today; I am closing this as done — please open a new thread if a chip of yours is not picked up.
